packages/uniforms-bootstrap4/src/SelectField.tsx

```diff
@@ -24,30 +24,28 @@ export type SelectFieldProps = HTMLFieldProps<
   }
 >;
 
-function Select({
-  allowedValues,
-  checkboxes,
-  className,
-  disabled,
-  error,
-  errorMessage,
-  fieldType,
-  id,
-  inline,
-  inputClassName,
-  inputRef,
-  label,
-  name,
-  onChange,
-  placeholder,
-  required,
-  showInlineError,
-  transform,
-  value,
-  ...props
-}: SelectFieldProps) {
+function Select(props: SelectFieldProps) {
+  const {
+    allowedValues,
+    checkboxes,
+    disabled,
+    error,
+    fieldType,
+    id,
+    inline,
+    inputClassName,
+    inputRef,
+    label,
+    name,
+    onChange,
+    placeholder,
+    required,
+    transform,
+    value,
+  } = props;
+
   return wrapField(
-    { ...props, id, label },
+    props,
```

Comment:
This means that the `wrapField` receives a lot of unnecessary props. It'd be much better to list (and pass here) only the ones that are actually needed.

I'm also quite sure that in order to fix #785, it'd be enough to remove line 30 (`className`) as it's not used anywhere. The same goes for #793 and line 44 (`showInlineError`).

It's weird that our ESLint did not catch that. Also, ping @kestarumper and @Monteth.

Reply:
wrapField requires not only these properties
```
className
showInlineError
```
as you can see from this list https://github.com/vazco/uniforms/blob/master/packages/uniforms-bootstrap4/src/wrapField.tsx#L26-L38

we should pass more properties such as:
```
className,
disabled,
error,
errorMessage,
required,
showInlineError
```

Reply:
Then these should be added here as well. It's definitely better to list them here rather than pass all.

Reply:
@radekmie  next props are used only in ``wrapField``
```
className
showInlineError
errorMessage
```
next props are used in ``wrapField`` and ``Select``
```
disabled,
error,
required
```